core/commands/add.go

```diff
@@ -48,6 +70,12 @@ const (
 	inlineOptionName      = "inline"
 	inlineLimitOptionName = "inline-limit"
 	toFilesOptionName     = "to-files"
+
+	preserveModeOptionName  = "preserve-mode"
+	preserveMtimeOptionName = "preserve-mtime"
+	modeOptionName          = "mode"
+	mtimeOptionName         = "mtime"
+	mtimeNsecsOptionName    = "mtime-nsecs"
```

Comment:
:thought_balloon:  Is it possible to preserve mtime without preserving mode?   

To  avoid cross-implementation issues, we should have "zero values" documented in https://github.com/ipfs/specs/pull/331/files